src/main/java/com/cactus/systems/GroceryListSystem.java

```diff
@@ -101,46 +33,30 @@ public void logout(){
      * @param name
      * @return true if a new groceryList was created, false otherwise
      */
-    public boolean newGroceryList(String name){
-        GroceryList newGroceryList = this.groceryListManager.createGroceryList(name, this.currentUser.getId());
+    public Response newGroceryList(String name, long userid){
```

Comment:
I'm wondering if it's necessary to return the entire Response object here. I don't think the grocery list id should be exposed to the UI. Also, I don't think the UI necessarily has access to the user ID, that would fall under the jurisdiction of the controller.

Reply:
sounds good